test/cppsuite/test_harness/workload/database_operation.cxx

```diff
@@ -85,6 +85,9 @@ database_operation::populate(
     /* Keys must be unique. */
     testutil_assert(key_count <= pow(10, key_size));
 
+    logger::log_msg(
```

Comment:
My stressful config has 500 collections, so this can take a while, having a pre collection creation message is helpful.